### PR TITLE
Workaround for no barometric sensor: intervals shows dashes (-) for gain and loss.

### PR DESCRIPTION
**Pull request by rgmf**
**Describe the pull request**
The simple solution: show dashes.
Hiding data require some communication strategy between Intervals Adapter and Interval Fragment because IntervalFragment have to hide the header and Adapter the values... Does it worth? I don't think so.

**Link to the the issue**
#791

### CODE REVIEW

File: src/main/java/de/dennisguse/opentracks/adapters/IntervalStatisticsAdapter.java [hunk 1]
```diff
@@ -59,8 +59,9 @@ public void onBindViewHolder(@NonNull RecyclerView.ViewHolder holder, int positi
 
         viewHolder.rate.setText(StringUtils.formatSpeed(context, interval.getSpeed(), metricUnits, isReportSpeed));
 
-        viewHolder.gain.setText(StringUtils.formatDistance(context, Distance.of(interval.getGain_m()), metricUnits));
-        viewHolder.loss.setText(StringUtils.formatDistance(context, Distance.of(interval.getLoss_m()), metricUnits));
+        viewHolder.gain.setText(interval.hasGain() ? StringUtils.formatDistance(context, Distance.of(interval.getGain_m()), metricUnits) : "-");
```
Comment (dennisguse) [inline src/main/java/de/dennisguse/opentracks/adapters/IntervalStatisticsAdapter.java:62]: Should be `R.strings.value_unknown` aka `<string name="value_unknown" translatable="false">-</string>`

**Review by dennisguse — CHANGES_REQUESTED** [review]
Still looks good

File: src/main/java/de/dennisguse/opentracks/adapters/IntervalStatisticsAdapter.java (hunk 1 above)
Comment (rgmf) [reply src/main/java/de/dennisguse/opentracks/adapters/IntervalStatisticsAdapter.java]: Done... I didn't realize of that.

**Review by dennisguse — APPROVED** [review]
+1

**Pull request merged**